docs/psa-config.md

```diff
@@ -1,16 +1,20 @@
-# Percona Scheduler Admin configuration file
+# Example of the Percona Scheduler Admin configuration file
 
 *ProxySQL* with the `percona-scheduler-admin` stores the parameters in a
 configuration file that uses the toml format. This file defines
 the server credentials and other settings.
 
 Use the `--config-file` option to run the *percona-scheduler-admin* script.
 
-**NOTE**: It is not recommended to store the scheduler config file in the Home directory.
+!!! caution
```

Comment:
Maybe we should say !!! warning here? To have the same style everywhere.

Reply:
Fixed.